command/meta_backend.go

```diff
@@ -1638,27 +1554,16 @@ Plan Serial:    %[1]d
 Current Serial: %[2]d
 `
 
-const inputBackendMigrateChange = `
-Would you like to copy the state from your prior backend %q to the
-newly configured %q backend? If you're reconfiguring the same backend,
-answering "yes" or "no" shouldn't make a difference. Please answer exactly
-"yes" or "no".
-`
-
-const inputBackendMigrateLegacy = `
-Terraform can copy the existing state in your legacy remote state
-backend to your newly configured backend. Please answer "yes" or "no".
+const outputBackendMigrateChange = `
+Terraform detected the backend type changed from %q to %q.
```

Comment:
I think "Terraform detected _that_ the..." would be clearer here. On first read my mental parser was confused by trying to read it as "Terraform detected the backend type".